structs.h

```diff
@@ -90,6 +92,7 @@ typedef struct ItemGetRecordStruct {
 	unsigned int dwTimestamp;
 } ItemGetRecordStruct;
 
+#pragma pack(push, 4)

```

Comment:
Is it still needed to change the default packing now that we are copying the individual structure members?

Reply:
Fixed - any save-related structs that don't have pointer members are still loaded with a single memcpy and so pragma pack is required, but it has been removed from structs that don't need it